Sources/Purchasing/Purchases/Attribution.swift

```diff
@@ -241,13 +241,14 @@ public extension Attribution {
 
     /**
      * Subscriber attribute associated with the OneSignal User ID for the user.
-     * Required for the RevenueCat OneSignal integration with versions v11.0 and above.
+     * Deprecated. Use `setOnesignalID` instead.
      *
      * #### Related Articles
      * - [OneSignal RevenueCat Integration](https://docs.revenuecat.com/docs/onesignal)
      *
      *- Parameter onesignalUserID: Empty String or `nil` will delete the subscriber attribute.
      */
+    @available(*, deprecated, message: "This API is not currently available. Contact support if you need to use it.")
```

Comment:
Nice 👍🏻 

You can add a reference to the other one too:
```suggestion
    @available(
      *, 
      deprecated,
      message: "This API is not currently available. Contact support if you need to use it.",
      renamed: "setOnesignalID"
      )
```